toon.py

```diff
@@ -81,12 +91,18 @@ class ToonDataStore:
     """An object to store the toon data."""
 
     def __init__(self, username, password, consumer_key, consumer_secret,
-                 gas=DEFAULT_GAS, solar=DEFAULT_SOLAR):
+                 gas=DEFAULT_GAS, solar=DEFAULT_SOLAR, tenant=DEFAULT_TENANT,
+                 name=DEFAULT_NAME):
         """Initialize toon."""
         from toonapilib import Toon
 
         # Creating the class
-        toon = Toon(username, password, consumer_key, consumer_secret)
+        toon = Toon(username,
+                    password,
+                    consumer_key,
+                    consumer_secret,
+                    tenant_id=tenant,
+                    display_common_name=name)
 
         self.toon = toon
         self.gas = gas
```

Comment:
What do these undocumented config values actually do?

Reply:
Sorry, i missed this! I have been quite busy and not paying the proper attention apparently. The tenant id is a value about the provider. For the Netherlands it is eneco but toon is a device by quby that is rebranded in other countries too so by being able to pass the tenant id explicitly this component works for the same device in spain too. As for the display_common_name it is the name of the device that can be retrieved by the app at "Your Toon" as display code and can be used by people with multiple devices(houses) on the same eneco contract.

Reply:
@Boltgolt do you want to have a look at the issue @Housebeer reports and if it is nothing I can merge?

Reply:
I was specifically asking about the "gas" and "solar" config options. As far as i can see they don't actually do anything?